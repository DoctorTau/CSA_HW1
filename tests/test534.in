583
8365 5637 -5510 589 9815 3392 -278 -1338 -4497 5755 -2231 9777 -4886 4866 -5527 -7637 7735 -5773 5424 2237 -3274 5198 1576 -1700 6641 7964 -4549 -9580 8410 7448 836 -6329 6976 -1066 -3862 5839 1399 -6903 -8016 -51 -2453 8405 -3694 -6393 -9680 8583 -6346 -443 4450 -6964 2771 -2735 8755 -7174 1596 8516 2670 -162 -7628 -8167 8002 9279 5781 5973 -7893 1959 7891 -9891 -4918 474 -7756 -8070 -3223 -7053 -8784 -6346 -871 -1735 2667 4502 5309 8191 -4472 7725 3492 102 -2544 8453 2005 -9220 545 -4731 8300 -2572 3700 7877 -3728 -1337 -8908 4261 -4542 8494 7117 5694 -8450 -6874 -4463 -2877 9344 -8914 -4986 -6700 -7522 -8960 3017 1686 -9088 -6678 3054 1315 -6119 -8398 3583 -1398 -3918 3309 -6025 9473 2017 4979 2964 9280 -9573 8844 4825 4620 -4933 7297 3900 3652 5651 -8304 1596 -4913 -8546 -424 -2285 -4800 9785 -6592 580 -9931 -9361 -6235 4108 -515 1434 -3371 -5546 1012 4027 6356 -1538 1264 -221 -939 9207 120 -3955 6280 -1450 -8023 8486 524 -8601 9044 1270 2351 -4409 -3320 9083 1612 -1813 -8653 -1069 -2151 -110 4596 -7330 866 3439 7571 1144 7680 -8235 -3148 9505 7515 3615 -2553 -794 -5153 -8949 3128 4120 2979 7458 -4124 2876 -5279 -9452 4182 -6484 2586 9506 -8960 1008 2971 -8520 -3777 6754 -239 -8683 5281 -5886 -5794 -4516 6301 -7745 6634 -2849 -1327 -6887 6967 -6021 -7597 -6154 -8197 -3252 3045 -7496 7909 -8891 -7704 -7608 7119 7251 -2794 -3951 1744 -7938 3672 -7352 -4687 -5347 988 7163 2271 7523 6336 2623 1457 5231 -2355 -1695 -2678 312 -4182 -6646 -8291 9575 -2314 -1543 -740 -5342 9258 9990 4620 9444 1686 3613 -247 -5748 -6636 -3414 -310 3677 -3539 8378 7882 -3960 -540 7050 -2057 -8745 -3086 5385 315 7046 -403 7349 -459 -1988 -6874 5902 -152 3848 5834 -3061 -4814 -3994 -6462 -1975 -1818 1981 -599 2464 -828 -8674 -9342 -5594 -8966 8127 5307 1782 -8705 4098 -912 5037 2822 -2166 -5300 3163 -7130 -1895 949 6781 -6182 -9820 -3472 -9171 -4773 5416 462 -4439 -5513 -7306 -625 7760 6859 -6547 -3427 1160 -7871 1088 6504 -5884 -2543 1203 1060 6142 -8418 6278 -765 3637 -9247 5690 3206 624 7143 5298 -3412 -8155 -3944 1774 833 277 8582 -4233 364 -5726 -5568 -3084 -4247 4992 -5597 -1692 -2593 7578 67 -2642 -1986 9010 979 -1753 4525 784 -821 -7772 -3354 72 5677 342 -8483 -3019 -7855 5326 709 -3721 2070 -6342 9467 6650 -3260 5407 -2072 1292 8915 6091 -1124 1014 9820 572 5117 -7853 1566 6762 -739 5416 9519 -3344 1336 -863 5697 -2907 -5071 -8908 1837 -5079 -1904 -8517 1726 -1667 -3111 -5283 4977 2422 8787 8630 -4524 8655 -6205 -2763 -7682 -3761 -7046 4104 -7513 9484 7168 6440 -3116 2463 7308 -7465 6800 5674 9150 7698 -3943 8871 462 61 -3120 8036 -5545 1401 1179 -399 2222 7468 -4050 -9798 -1453 6996 -9608 -8293 -9704 4941 -1495 -4466 9721 -1419 -459 1142 311 3216 -5639 8248 3138 6311 -32 -3640 -610 -7492 2649 5452 8171 7162 1459 -6735 -3511 599 590 9299 1457 9450 -5959 -6555 8064 8175 8315 -2763 1815 -5823 -9559 9639 -3427 -3075 -3455 -8325 -282 -8296 -7598 5602 -8035 3725 -8691 5944 6027 376 -1623 3157 2828 847 -653 4527 5317 -9320 3206 -914 -2484 -6553 7381 9998 7476 4273 -4136 -1621 -1026 6008 7048 4985 -6835 -1447 -4402 -9996 -9207 5369 -4117 -5250 4144 5198 -3975 -6455 -6136 -9937 883 9682 -3520 3636 -9179 7907 