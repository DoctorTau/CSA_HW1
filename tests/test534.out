-5510 -278 -1338 -4886 -3274 -1700 -9580 -1066 -3862 -8016 -3694 -9680 -6346 -6964 -7174 -162 -7628 -4918 -7756 -8070 -8784 -6346 -4472 -2544 -9220 -2572 -3728 -8908 -4542 -8450 -6874 -8914 -4986 -6700 -7522 -8960 -9088 -6678 -8398 -1398 -3918 -8304 -8546 -424 -4800 -6592 -5546 -1538 -1450 -3320 -110 -7330 -3148 -794 -4124 -9452 -6484 -8960 -8520 -5886 -5794 -4516 -6154 -3252 -7496 -7704 -7608 -2794 -7938 -7352 -2678 -4182 -6646 -2314 -740 -5342 -5748 -6636 -3414 -310 -3960 -540 -3086 -1988 -6874 -152 -4814 -3994 -6462 -1818 -828 -8674 -9342 -5594 -8966 -912 -2166 -5300 -7130 -6182 -9820 -3472 -7306 -5884 -8418 -3412 -3944 -5726 -5568 -3084 -1692 -2642 -1986 -7772 -3354 -6342 -3260 -2072 -1124 -3344 -8908 -1904 -4524 -7682 -7046 -3116 -3120 -4050 -9798 -9608 -9704 -4466 -32 -3640 -610 -7492 -282 -8296 -7598 -9320 -914 -2484 -4136 -1026 -4402 -9996 -5250 -6136 -3520 